lib/ansible/module_utils/k8s/common.py

```diff
@@ -225,12 +220,12 @@ def load_resource_definitions(self, src):
 
     @staticmethod
     def diff_objects(existing, new):
-        if not HAS_DICTDIFFER:
-            return False, []
-
-        diffs = list(dictdiffer.diff(new, existing))
-        match = len(diffs) == 0
-        return match, diffs
+        result = dict()
```

Comment:
Is the output identical? Otherwise changing this would break backwards compatibility for 3rd party modules.

Reply:
The output is not identical, but the current output is almost unreadable for a human.

The only consumer in Ansible's code base is the k8s/raw module_util, which is consumed by the k8s module. I'm not too concerned about breaking backward compatibility for a diff output but perhaps I should be.